mwdb/web/src/components/Navigation.js

```diff
@@ -22,40 +21,29 @@ import logo from "../assets/logo.png";
 
 function AdminNav() {
     const auth = useContext(AuthContext);
-    const [pendingUsersCount, setPendingUsersCount] = useState(null);
-
+    const config = useContext(ConfigContext);
     const isAdmin = auth.isAdmin;
 
-    async function updatePendingUsersCount() {
-        try {
-            let response = await api.getPendingUsers();
-            setPendingUsersCount(response.data.users.length);
-        } catch (error) {
-            console.error(error);
-            setPendingUsersCount("?");
-        }
-    }
-
     useEffect(() => {
         if (!isAdmin) return;
-        let timer = setInterval(updatePendingUsersCount, 15000);
-        updatePendingUsersCount();
+        let timer = setInterval(config.getPendingUsers, 15000);
+        config.getPendingUsers();
         return () => {
             clearInterval(timer);
         };
-    }, [isAdmin]);
+    }, [isAdmin, config]);
```

Comment:
Maybe it would be better to move that timer into ConfigProvider? Currently it's defined in two places.

It can be registered via useEffect. Also remember that pending users should be updated only if registration features are enabled `config.is_registration_enabled` and user is an admin.